pkg/driver/clients/http_client.go

```diff
@@ -48,18 +99,31 @@ func (i *httpInvoker) Invoke(function *common.Function, runtimeSpec *common.Runt
 	////////////////////////////////////
 	// INVOKE FUNCTION
 	////////////////////////////////////
-	start := time.Now()
-	record.StartTime = start.UnixMicro()
 
 	requestBody := &bytes.Buffer{}
 	/*if body := composeDandelionMatMulBody(function.Name); isDandelion && body != nil {
 		requestBody = body
 	}*/
+	contentType := "application/octet-stream"
```

Comment:
You already got this as a global variable.

Reply:
Fixed in https://github.com/vhive-serverless/invitro/pull/606/commits/7f77193a5f3e7523fc6ed667a2836e1aa409285c.